lib/rspec/support/object_formatter.rb

```diff
@@ -16,6 +16,10 @@ def self.default_instance
         @default_instance ||= new
       end
 
+      def self.default_max_formatted_output_length=(length)
+        default_instance.max_formatted_output_length = length
```

Comment:
Totally. I did this to have a shorter call chain over on the Expectations side. Do you think this is fine and we can just merge?